include/clientsubmit.php

```diff
@@ -14,21 +14,22 @@
   PURPOSE. See the above copyright notices for more information.
 =========================================================================*/
 
+use CDash\Config;
+use CDash\Model\ClientSite;
+use CDash\Model\ClientOS;
+use CDash\Model\ClientJobSchedule;
+use CDash\Model\ClientCompiler;
+use CDash\Model\ClientCMake;
+use CDash\Model\ClientLibrary;
+use CDash\Model\ClientJob;
+
 function client_submit()
 {
-    include 'config/config.php';
```

Comment:
??? this diff looks funny to me

Reply:
Yeah, that is funky, but I think it's correct.